generic_mux.go

```diff
@@ -11,7 +11,7 @@ type Registerer[T, B any] interface {
 func Registers[B, T any](engine *Engine, a Registerer[B, T]) *Route[B, T] {
 	route := a.Register()
 
-	err := route.RegisterOpenAPIOperation(engine.OpenAPI)
+	err := route.RegisterOpenAPIOperation(engine.OpenAPI, engine.OpenAPIConfig)
```

Comment:
🤔 

I really like this signature. But part of me for future compatibility purpose thinks we should just pass the whole engine.

What are others thought?